### ISSUE TITLE
Suggestion for french translation correction

### ISSUE THREAD

**Issue by Siltaar** [cryptocat/cryptocat#708]
Currently, the french translation shows: État de la conversation: cryptée.
A correct translation is: État de la conversation : chiffrée.
In a nutshell, we have : 'chiffrer' (to encrypt/cipher), 'déchiffrer' (decipher) and 'decrypter' (decipher without the key, or explain complicated things).

In addition, the correct translation is already used in the label : Send a ciphered file / Envoyer un fichier chiffré.

**Comment by Jiehong** [comment]
+1

**Comment by DGO143** [comment]
https://github.com/cryptocat/cryptocat/blob/master/src/core/locale/fr.txt
line 85 should be chiffrée?

**Comment by DGO143** [comment]
What should line 86 be?

**Comment by Jiehong** [comment]
line 86 should be `non chiffrée !`. 

Some other lines need to be changed, so I'll send you a pull request, it will be easier that way ;)

**Comment by Jiehong** [comment]
Ok, I'll use Transifex, because I've just seen that it's supposed to be done that way.

**Comment by Jiehong** [comment]
It seems that the transifex activity isn't very high recently, and I'm not even sure when I'll be accepted as part of the French team.

Should I perform a pull-request instead?

**Comment by DGO143** [comment]
I'm not sure if this is correct in context, `...n'est pas chiffree` would be very difficult to do.

**Comment by DGO143** [comment]
>Some other lines need to be changed, so I'll send you a pull request, it will be easier that way ;)
Good idea, please put `#708` in the pull request so it shows in this issue.
